src/ontology/foodon-edit.owl

```diff
@@ -65857,17 +65872,22 @@ SubClassOf(obo:FOODON_03530217 obo:FOODON_03530008)
 
 # Class: obo:FOODON_03530218 (food packing medium integrity)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "It is important to distinguish whether the food in question has been drained or not drained when evaluating the nutrient content.
-Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530218 "A can of peaches in syrup, in which the syrup is uncompromised."@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530218 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530218 "A food object quality which describes the integrity of the medium in which the food is packed for preservation and handling or the medium surrounding foods."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03530218 "Existing definition was a legacy definition, not ontological and has been moved to comments. Updated examples of usage to remove plastic usage as describes packaging not packing medium."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530218 "http://langual.org")
 AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03530218 "http://www.langual.org/langual_thesaurus.asp?termid=Z0218")
+AnnotationAssertion(rdfs:comment obo:FOODON_03530218 "Previous definition: It is important to distinguish whether the food in question has been drained or not drained when evaluating the nutrient content.
+Related term to PRESENCE OF PACKING MEDIUM is K.  PACKING MEDIUM."@en)
 AnnotationAssertion(rdfs:label obo:FOODON_03530218 "food packing medium integrity"@en)
 SubClassOf(obo:FOODON_03530218 obo:FOODON_00002470)
 
 # Class: obo:FOODON_03530219 (food packing medium drained)
 
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530219 "A can of chickpeas where the liquid medium has been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530219 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530219 "A food object quality where the medium in which the food is packed is drained."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
```

Comment:
Add yourself to the edit history with a term editor annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03530219 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530219 <http://orcid.org/0000-0001-8008-8249>)
```

Reply:
Resolved in later commits.